### ISSUE TITLE
DataForm JSON validation

### ISSUE THREAD

**Issue by davorpeic** [telerik/nativescript-ui-feedback#201]
Hi, 

I'm having issues enabling validation for the form defined in JSON, is it possible to add validation rules in json? In html template (angular) work fine as there are tags added, but can't get it to work in json, all examples point to editors defined in html.

TNS 3 + Angular

thanks

**Comment by tgpetrov** [comment]
Hi @davorpeic,

Defining what validators to be used through JSON is not supported. You will need to use the html as the examples show.

Thank you for the suggestion. We will consider it for our future releases.

**Comment by davorpeic** [comment]
Ok, no problem, just wanted to be sure I'm not doing something wrong.

thanks

**Comment by VladimirAmiorkov** [comment]
@davorpeic I would like to reopen this issue as it is a feature that we will discuss internally when we can implement it.

**Comment by davorpeic** [comment]
Any idea why conditional on validation tag would not work? I'm trying to do a workaround to show validation only if there is one from my json, but it looks like `*ngIf` is not working.. this is how I try to use it

```
<TKMinimumLengthValidator *ngIf="{{item.minLength}}" tkEntityPropertyValidators errorMessage="Password must be at least 6 characters long." length="6"></TKMinimumLengthValidator>
```

thanks

**Comment by Natalia-Hristova** [comment]
Hello @davorpeic ,

Some update about this feature - now it's available in the latest release v 3.1.0. Read more [here](http://docs.telerik.com/devtools/nativescript-ui/Controls/NativeScript/DataForm/Validation/dataform-validation-overview#json-metadata).

**Comment by davorpeic** [comment]
@Natalia-Hristova OMG Natalia, this looks awesome, will check it soon!
